meta-iot2000-example/scripts/lib/wic/canned-wks/example-image.iot2000.wks

```diff
@@ -1,3 +1,14 @@
+# short-description: Create an Galileo Gen 1/2 disk image
+# long-description: Creates a partitioned EFI disk image for Intel Galileo Gen 1/2,
+# using efibootguard, that the user can directly dd to boot media.
+
+part --source efibootguard-efi --size 32 --extra-space 0 --overhead-factor 1 --ondisk mmcblk0 --label efi --part-type=EF00 --align 1024
+part / --source rootfs --size 1536 --extra-space 0 --overhead-factor 1 --ondisk mmcblk0 --fstype=ext4 --label platform0 --align 1024
+part --source rootfs --size 1536 --extra-space 0 --overhead-factor 1 --ondisk mmcblk0 --fstype=ext4 --label plaftorm1 --align 1024
+part --source efibootguard-boot --size 32 --extra-space 0 --overhead-factor 1 --ondisk mmcblk0 --label boot0 --align 1024 --part-type=0700 --sourceparams "watchdog=60,revision=2"
+part --source efibootguard-boot --size 32 --extra-space 0 --overhead-factor 1 --ondisk mmcblk0 --label boot1 --align 1024 --part-type=0700 --sourceparams "watchdog=60,revision=1"
 part swap --ondisk mmcblk0 --size 512 --fstype=swap --label swap --align 1024
+part --size 1024 --extra-space 0 --overhead-factor 1 --ondisk mmcblk0 --label persistent --align 1024 --fstype=ext4
+
+bootloader --ptable gpt --append="console=ttyS1,115200n8 earlyprintk"
 
-include common.iot2000.wks.inc
```

Comment:
And now we are missing the initrd with ACPI fixups. While I hope we can overcome that soon, we require it for now. I suppose that explains why you patches didn't need some of the fixes of Farhat...